practice.css

```diff
@@ -1,3 +1,36 @@
-.details:checked + .label{
```

Comment:
Give appropriate file names instead of practice.js/css/html. Check file naming conventions and use the appropriate one.